cmd/transportRequestCreateCTS.go

```diff
@@ -0,0 +1,88 @@
+package cmd
+
+import (
+	"github.com/SAP/jenkins-library/pkg/command"
+	"github.com/SAP/jenkins-library/pkg/log"
+	"github.com/SAP/jenkins-library/pkg/telemetry"
+	"github.com/SAP/jenkins-library/pkg/transportrequest/cts"
+)
+
+type transportRequestCreateCTSUtils interface {
+	command.ExecRunner
```

Comment:
use the `type Exec interface `from create.go

Reply:
That is generated code. Why should we change it? It looks like that in almost all the steps and I guess we are more familiar with that. Having "our" interface here would also work, of course.